script/Arcueid/Day 3/03_00_ARC03_1_1.txt

```diff
@@ -67,7 +72,8 @@ Having breakfast after seven o'clock is a sign you're getting soft."
 [sha:af4ec8dd210df44b106d4b0e5713e641f4cd0d1a]{
 -- Page 38, Offset 4029.
 -- いや７時はフツーだろー、と言いたい気持ちをぐっと堪える。この家はそもそもフツーではない。
-I resist the urge to say, 'No, seven o'clock is normal'. It's this household that's abnormal.
+I resist the urge to point out that seven o'clock is a normal time to have breakfast.
```

Comment:
i think this version is a little overwritten and the prev. version was fine. revert